packages/neuron-wallet/src/controllers/app/index.ts

```diff
@@ -155,6 +162,10 @@ export default class AppController {
     AppController.showMessageBox(options)
   }
 
+  public static checkUpdates(menuItem: MenuItem) {
+    UpdateController.checkUpdates(menuItem)
```

Comment:
The name `update controller checks updates menu item` is confusing.

Reply:
Indeed. What's your suggestion? We do need to pass the menu item for disabling/enabling.

Reply:
`checkUpdates(sender: MenuItem)`, `checkUpdates(sender: any)`?

Reply:
If [Menu.getApplicationMenu](https://electronjs.org/docs/api/menu#menugetapplicationmenu) works in the update controller, the `menuitem` parameter is redundant.

Reply:
We might trigger this by other UI component, eg from a button. `sender: any` is a common signature when programming macOS/iOS app.

Reply:
> If [Menu.getApplicationMenu](https://electronjs.org/docs/api/menu#menugetapplicationmenu) works in the update controller, the `menuitem` parameter is redundant.

Not exactly. We can't be sure the menu item triggering it is from the main menu. (But if we have multiple entries then we do need to disable/enable all).

I'd say we leave this as. If we add more action entries then we refactor/rename this.

Reply:
LGTM